src/hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useState } from 'react';
+
+type LocalStorageKey = 'random-pick-numbers' | 'random-pick-names';
```

Comment:
```suggestion
const LOCAL_STORAGE_KEY = {
  randomPickNumbers: 'randomPickNumbers',
  randomPickNames: 'randomPickNames',
} as const;

type LocalStorageKey = typeof LOCAL_STORAGE_KEY[keyof typeof LOCAL_STORAGE_KEY];
```
이렇게 키를 상수화해서 constant 파일로 분리하고, 타입을 가져다 사용하는 건 어떠신가요? `useLocalStorage` 훅은 특정 관심사에 구애받지 않고 어디서나 사용 가능한 훅인 것 같아서, 관심사를 갖는 코드를 분리해내면 좋겠다는 생각이 드네요. ㅎㅎ